primitiv/operator_impl.cc

```diff
@@ -576,9 +672,15 @@ BACKWARD(SparseSoftmaxCrossEntropy) {
   gy.device().pick_bw(-gy, ids_, dim_, *gx[0]);
 }
 
-BACKWARD(StopGradient) {}
+BACKWARD(StopGradient) {
+  UNUSED(y);
+  UNUSED(gy);
+  UNUSED(x);
+  UNUSED(gx);
+}
 
 #undef BACKWARD
+#undef UNPUSED
```

Comment:
Is this a typo?
I think `#undef UNUSED` is correct.

Reply:
Fixed.